packages/boa/package.json

```diff
@@ -5,14 +5,15 @@
   "main": "lib/index.js",
   "scripts": {
     "clean": "sh tools/clean-python.sh",
-    "preinstall": "node tools/install-python.js && node tools/install-requirements.js",
+    "preinstall": "make -C ./pybind11/ && node tools/install-python.js && node tools/install-requirements.js",
     "postinstall": "npm run build",
     "test": "tap --no-cov ./tests/**/*.js",
     "lint": "eslint . && ./clang-format.py",
     "build": "node-gyp configure && node-gyp build",
     "pretest": "npm run lint",
     "codecov": "nyc report --reporter=lcovonly && codecov",
-    "htmlcov": "nyc report --reporter=html"
+    "htmlcov": "nyc report --reporter=html",
+    "compile": "npm run build"
```

Comment:
Why not `npm run build` directly?

Reply:
`npm run build` can't trigger to compile.